OffbeatStore/utility_functions/operations.py

```diff
@@ -50,6 +50,9 @@ def delete_product_operation(id):
     query = "DELETE FROM reaction WHERE product_id = ?"
     db.execute(query, (id,))
     db.commit()
+    query = "DELETE FROM notification WHERE product_id = ?"
+    db.execute(query, (id,))
```

Comment:
We can do this in a better way by adding constraints and relationships between tables/entities.
Not blocking now. We will talk about this separately.

Quick links:
- https://www.w3schools.com/sql/sql_constraints.asp
- https://www.techonthenet.com/sqlite/foreign_keys/foreign_delete.php